aws-lc-sys/builder/main.rs

```diff
@@ -455,3 +487,130 @@ fn setup_include_paths(out_dir: &Path, manifest_dir: &Path) -> PathBuf {
 
     include_dir
 }
+
+#[derive(Default)]
+#[allow(dead_code)]
+pub(crate) struct BindingOptions {
+    pub build_prefix: Option<String>,
+    pub include_ssl: bool,
+    pub disable_prelude: bool,
+}
+
+fn invoke_external_bindgen(
+    manifest_dir: &Path,
+    prefix: &Option<String>,
+    gen_bindings_path: &Path,
+) -> Result<(), String> {
+    if !test_command("bindgen".as_ref(), &["--version".as_ref()]).status {
+        return Err("External bindgen command not found.".to_string());
+    }
+    let options = BindingOptions {
+        build_prefix: None,
+        include_ssl: false,
+        disable_prelude: true,
+    };
+
+    let clang_args = prepare_clang_args(manifest_dir, &options);
+    let header = get_rust_include_path(manifest_dir)
+        .join("rust_wrapper.h")
+        .display()
+        .to_string();
+
+    let sym_prefix: String;
+    let mut bindgen_params = vec![];
+    if let Some(prefix_str) = prefix {
+        sym_prefix = if target_os().to_lowercase() == "macos" || target_os().to_lowercase() == "ios"
+        {
+            format!("_{prefix_str}_")
+        } else {
+            format!("{prefix_str}_")
+        };
+        bindgen_params.extend(vec!["--prefix-link-name", sym_prefix.as_str()]);
+    }
+
+    bindgen_params.extend(vec![
```

Comment:
Can you leave a comment that these need to be kept in-sync with the options specified with the bindgen library. Specifically items like the allowed files allowlist, and the rustified-enums etc.